docs/connecting-to-ethereum/adding-members.md

```diff
@@ -0,0 +1,45 @@
+---
+id: adding-members
+title: Adding members
+---
+
+With multi-signature orgs, you can add multiple **members** who can also initiate transactions such as registrations or anchors in your org. If you created a single-signer org, you can not add more members. Read more about creating an org [here][co].
+
+## Adding a member
+
+To start, navigate to the **Members** tab in your org page.
+
+![single member org page][om]
+
+To add members, you must do so in your Gnosis Safe. Clicking **Add a member** will lead you to your Safe. 
+
+![single member button press][bp]
+
+If you haven't already, connect your wallet to the Gnosis Safe UI. Be sure to use the one that is linked to your Radicle identity. Click on the **Settings** tab and navigate to *Owners*.
+
+![gnosis safe ui owners][gso]
+
+Here, you can add additional owners and update your transaction threshold. This is number of owners needed to sign-off on transactions within the Gnosis Safe. This will be the number of people required to sign off on any anchor or other org interaction. Follow the instructions outlined in the [Gnosis Safe docs][gsd] if you need help.
```

Comment:
```suggestion
Here, you can add additional owners and update your quorum threshold. This is number of owners needed to sign-off on transactions within the Gnosis Safe. This will be the number of people required to sign off on any anchor or other org interaction. Follow the instructions outlined in the [Gnosis Safe docs][gsd] if you need help.
```